ClientApp/viking-vault/src/UserDashboard/AutomaticPaymentList.tsx

```diff
@@ -90,7 +101,7 @@ class AutomaticPaymentList extends React.Component<any, IAutomaticPaymentsState>
                 <td className="payments-text centered-text">{this.formatDate(payment.lastPaymentDate)}</td>
                 <td className="payments-text centered-text"><ActivatePaymentToggle paymentId={payment.id} /></td>
                 <td className="payments-text centered-text">Edit element here</td>
-                <td className="payments-text centered-text">Delete element here</td>
+                <td><DeleteAutomaticPayment automaticPaymentId = {payment.id} deletePaymentFromList = {this.deletePaymentFromList}/></td>
```

Comment:
Don't make the code hard to read just because you want to keep things in one line, in this case I think it will be better from a code readability standpoint if you'd move `<DeleteAutomaticPayment` on its own line